### DISCUSSION TITLE
Github 2FA not working!

### DISCUSSION THREAD

**Discussion by chamanbravo** [community/community#108391] [Code Security]
### Github 2FA not working! 

I tried setting up 2FA for github account. Now it prompts with verify 2FA.

![image](https://github.com/community/community/assets/76839908/0cd469ac-d070-4828-af65-6fc68af00780)
![image](https://github.com/community/community/assets/76839908/70840358-077d-4526-a16d-c6b36e438cee)
![image](https://github.com/community/community/assets/76839908/48feb4e2-acfe-44fb-bfdf-9b981bc3b10c)

But I don't receive any code in github mobile. Even authenticator doesn't work. I am daily prompted this. For now I just click verify tomorrow. I guess I'll get banned if this continues.

**Comment by kaliif** [comment]
Same here. I have configured Microsoft authenticator, Github mobile, and text message as my OTP endpoints but no codes are ever received. Nothing wrong with the apps, they work for the accounts, it's just Github that's broken.

Already was blocked once and had to use a recovery code. Any chance 2FA could be fixed?
